react-app/src/components/FilterTabs/FilterTabs.tsx

```diff
@@ -34,6 +34,7 @@ const FilterTabItem: <T>(props: FilterTabItemProps<T>) => React.ReactElement = (
         "border-b-2",
         "py-4",
         "px-1",
+        "w-20",
```

Comment:
where is this from?

Reply:
"All" is too short :(
This makes all the tab buttons same length in width

Reply:
I think that's fine 
the tab style we are using allows this
https://tailwindui.com/components/application-ui/navigation/tabs